Python/Product/PythonTools/PythonTools/Environments/VirtualEnv.cs

```diff
@@ -162,6 +190,13 @@ private static async Task ContinueCreate(IServiceProvider provider, IPythonInter
                 }
             }
 
+            try {
+                if (!useVEnv) {
```

Comment:
I think we might need to ask the user here? We're running something on their behalf in their global python.

Reply:
ah true. i think there is an example of this.. i'll try to find where we do something similar

Reply:
I think i'll try to detect this error case instead and just print this command for the user to run themselves